src/DotVVM.Framework/Resources/Scripts/DotVVM.ts

```diff
@@ -1408,7 +1408,7 @@ class DotVVM {
                     if (!isInitial) {
                         ko.virtualElements.setDomNodeChildren(element, ko.utils.cloneNodes(savedNodes!));
                     }
-                    ko.applyBindingsToDescendants(makeContextCallback(bindingContext, rawValue), element);
+                    ko.applyBindingsToDescendants(makeContextCallback(bindingContext, rawValue, _allBindings), element);
```

Comment:
`_allBindings` was named with the underscore since the parameter was unused. As we use it now, the name should not start with underscore.